Wikipedia/Code/ReadingThemesControlsViewController.swift

```diff
@@ -79,6 +79,25 @@ open class ReadingThemesControlsViewController: UIViewController, AnalyticsConte
         NSObject.cancelPreviousPerformRequests(withTarget: self)
     }
     
+    open override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        let tap = UITapGestureRecognizer(target: self, action: #selector(self.handleTap(_:)))
+        slider.addGestureRecognizer(tap)
+    }
+    
+    func handleTap(_ sender: UIGestureRecognizer) {
+        let pointTapped: CGPoint = sender.location(in: slider)
+        
+        let widthOfSlider: CGFloat = slider.bounds.size.width
+        let newValue = pointTapped.x * (CGFloat(slider.numberOfSteps) / widthOfSlider)
+
+        slider.value = Int(newValue)
```

Comment:
@nambatee ah, i should have tested - looks like it works fine without `rounded()`